quickwit-config/src/config.rs

```diff
@@ -56,7 +57,7 @@ fn default_metastore_uri(data_dir_path: &Path) -> Uri {
 }
 
 // See comment above.
-fn default_index_root_uri(data_dir_path: &Path) -> Uri {
+fn _default_index_root_uri(data_dir_path: &Path) -> Uri {
```

Comment:
Remove then?

Reply:
This is still being used in `QuickwitConfig`. I changed the name to differentiate the two. I can change it to something else if you have any recommendations.